README.MD

```diff
@@ -0,0 +1,157 @@
+integration code for the platform.
+
+The [Java and the Blockchain](https://www.youtube.com/watch?v=ea3miXs_P6Y) talk provides
+an overview of blockchain, Ethereum and Web3j.
+
+Features
+--------
+
+-   Complete implementation of Ethereum's
+    [JSON-RPC](https://github.com/ethereum/wiki/wiki/JSON-RPC) client
+    API over HTTP and IPC
+-   Ethereum wallet support
+-   Auto-generation of Java smart contract wrappers to create, deploy,
+    transact with and call smart contracts from native Java code
+    ([Solidity](http://solidity.readthedocs.io/en/latest/using-the-compiler.html#using-the-commandline-compiler)
+    and
+    [Truffle](https://github.com/trufflesuite/truffle-contract-schema)
+    definition formats supported)
+-   Reactive-functional API for working with filters
+-   [Ethereum Name Service (ENS)](https://ens.domains/) support
+-   Support for Parity's
+    [Personal](https://github.com/paritytech/parity/wiki/JSONRPC-personal-module),
+    and Geth's
+    [Personal](https://github.com/ethereum/go-ethereum/wiki/Management-APIs#personal)
+    client APIs
+-   Support for [Infura](https://infura.io/), so you don't have to run
+    an Ethereum client yourself
+-   Comprehensive integration tests demonstrating a number of the above
+    scenarios
+-   Command line tools
+-   Android compatible
+-   Support for JP Morgan's Quorum via
+    [web3j-quorum](https://github.com/web3j/quorum)
+-   Support for [EEA Privacy features as described in EEA
+    documentation](https://entethalliance.org/technical-documents/) and
+    implemented in [Hyperledger
+    Besu](https://besu.hyperledger.org/en/latest/Reference/API-Methods/#eea-methods).
+
+It has five runtime dependencies:
+
+-   [RxJava](https://github.com/ReactiveX/RxJava) for its
+    reactive-functional API
+-   [OKHttp](https://hc.apache.org/httpcomponents-client-ga/index.html)
+    for HTTP connections
+-   [Jackson Core](https://github.com/FasterXML/jackson-core) for fast
+    JSON serialisation/deserialization
+-   [Bouncy Castle](https://www.bouncycastle.org/) ([Spongy
+    Castle](https://rtyley.github.io/spongycastle/) on Android) for
+    crypto
+-   [Jnr-unixsocket](https://github.com/jnr/jnr-unixsocket) for \*nix
+    IPC (not available on Android)
+
+It also uses [JavaPoet](https://github.com/square/javapoet) for
+generating smart contract wrappers.
+
+QuickStart
+---------
+- Use Epirus CLI (recommended for full project creation)
+    - Install Epirus CLI (include Windows installation instructions too)
+    - Run ```$ epirus new```
+    - Run app:
+        - Create [Epirus account](https://docs.epirus.io/sdk/cli/#account-creation)
+        - Provide parameters to run project locally
+        
+- Use [Maven](https://github.com/web3j/web3j-maven-plugin) / [Gradle](https://github.com/web3j/web3j-gradle-plugin) plugin
+
+Documentation
+-------------
+Please head to the [Web3j Documentation](https://docs.web3j.io) for further instructions on using Web3j.
```

Comment:
Why not put this sentence like this:

### Please head to the [Web3j Documentation](https://docs.web3j.io) for further instructions on using Web3j.

without specifying that `Documentation` title.
Titles in md files can be done using hashtags... one hashtag is biggest title, two hashtags is smaller one etc. I think it goes down to 6 levels